### DISCUSSION TITLE
Problem running Nuxt on Macbook

### DISCUSSION THREAD

**Discussion by alkstal** [nuxt/nuxt#21782] [Questions]
I've had this problem with our Nuxt application and I've seen a couple of people mention the same problem on the Nuxt Discord but without any answers.

When running `npm run dev` this happens

```
No loader is configured for ".node" files: node_modules/fsevents/fsevents.node
[unhandledRejection] Build failed with 1 error:  
```

Here's a screenshot from another person from the Discord server,

![image](https://github.com/nuxt/nuxt/assets/70524463/c5681ceb-bd2d-4d0c-a2b2-80586a5f4d89)

This happens on my Macbook M2. No issue on Windows which is my normal development environment.
Nuxt version doesn't seem to matter. Happens on the latest version and older.

I'm clueless on what the issue might be.

Does anyone have any clue what might be causing this for some people?

**Comment by Kiansa** [comment]
This is just a wild guess, but sometimes `npx nuxi upgrade --force` doesn't really refresh node_modules and package-lock.json due to system permission. 
if that's the case, you need to delete `node_modules` and `package-lock.json` manually and then `npm i`.

**Reply by johanalkstal** [reply]
Thanks for the suggestion. Alas this is something I've already tried. The problem also happens on a fresh git repo without prior installation.
